File: CPP/Ram.cpp
class Ram
{
    private:
        /* data */
        string capacity;
        int price;

    public:
        Ram(/* args */) {
        }
        Ram(string m, int y) {
            this->capacity = m;
            this->price = y;
        }

        void setCapacity(string m) { this->capacity = m; }
        void setPrice(int y) { this->price = y; }

        string getCapacity() { return this->capacity; }
        int getPrice() { return this->price; }

        void printDetRam() {
            cout << "----------------------" << endl;
            cout << "Ram Capacity   : " << this->capacity << endl;
            cout << "Price          : " << this->price << endl;
        }

        ~Ram() {
        }
};

File: PHP/Ram.php
<?php

class Ram{
    private $capacity;
    private $price;

    public function __construct($c, $p){
        $this->capacity = $c;
        $this->price = $p;
    }

    public function setCapacity($c){
        $this->capacity = $c;
    }
    public function getCapacity(){
        return $this->capacity;
    }

    public function setPrice($p){
        $this->price = $p;
    }
    public function getPrice(){
        return $this->price;
    }

    public function printRam(){
        echo "---------------------- <br>";
        echo "Ram Capacity   : ". $this->capacity ."<br>";
        echo "Price          : ". $this->price ."<br>";
    }
    public function __destruct(){ }
}

?>
